x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/datafeed/DatafeedJob.java

```diff
@@ -318,7 +320,7 @@ private void run(long start, long end, FlushJobAction.Request flushRequest) thro
             try {
                 extractedData = dataExtractor.next();
             } catch (Exception e) {
-                LOGGER.debug("[" + jobId + "] error while extracting data", e);
+                LOGGER.error("[" + jobId + "] error while extracting data", e);
```

Comment:
```suggestion
                LOGGER.error(new ParameterizedMessage("[{}] error while extracting data", jobId), e);
```
Great to log the full trace now!!!